assets/src/scss/layout/_breadcrumbs.scss

```diff
@@ -26,6 +30,10 @@
       }
     }
     display: inline-block;
+
+    &:not(:only-child):not(:last-child):not(.page-type) {
+      margin-inline-end: $sp-1;
+    }
```

Comment:
Do you think this can be reversed? Otherwise it may be difficult for child theme to override `.tag-item` spacing. 

So define a default `margin-inline-end` and then override it for:
````
&:only-child,
&:last-child,
&:page-type
```

Reply:
This is actually just adding [the lines that were there before](https://github.com/greenpeace/planet4-master-theme/commit/18776d064d5c704ad0ed0caa770c967bdc5a7301#diff-edf8cadc9b9e8d401ce446b027742bd8a026dc2b6b42064f3d596935e541c875L29-L31), so I think it was like this for quite a while 🤔

Reply:
Ah ok then, so we know that child theme overrides won't break :+1: